JavaBasicProgram/src/StringMethod/stringExampleMethod.java

```diff
@@ -0,0 +1,90 @@
+package StringMethod;
+
+public class stringExampleMethod {
+    public static void main(String[] args) {
+        // TODO Auto-generated method stub
+        //charAt
+        String string = "Excellarate";
+        String firststring = "Excellarate";
+        String secondstring = "EXCELLARATE";
+        String thirdstring = "Enchora";
+        char result = string.charAt(0);//specified char index
+        System.out.println(result);
```

Comment:
create a separate method and call it from main method.

Reply:
creating separate method and call in main program.